recipe/Dockerfile

```diff
@@ -36,14 +37,14 @@ RUN apt-get --yes -qq update \
  && apt-get --yes -qq clean \
  && rm -rf /var/lib/apt/lists/*
 
-ENV GOSU_VERSION 1.11
+ENV GOSU_VERSION 1.17
```

Comment:
This change somehow leads to the following error with `Docker version 26.0.0, build 2ae903e`:
```
> [lab  3/11] RUN dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')"  && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/1.17/gosu-$dpkgArch"  && chmod +x /usr/local/bin/gosu  && gosu nobody true:                                                                      
0.339 --2024-04-12 02:32:35--  https://github.com/tianon/gosu/releases/download/1.17/gosu-amd64         
0.385 Resolving github.com (github.com)... 20.27.177.113, ::ffff:20.27.177.113                          
0.413 Connecting to github.com (github.com)|20.27.177.113|:443... connected.
0.479 HTTP request sent, awaiting response... 302 Found
0.729 Location: https://objects.githubusercontent.com/github-production-release-asset-2e65be/19708981/e0acf967-faba-4c34-a6ab-f544302e073b?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240412%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240412T023235Z&X-Amz-Expires=300&X-Amz-Signature=913fe54621644d5423990d67bad1b37303bdc302c667f2b9327fa3056c0d3bcf&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=19708981&response-content-disposition=attachment%3B%20filename%3Dgosu-amd64&response-content-type=application%2Foctet-stream [following]
0.729 --2024-04-12 02:32:35--  https://objects.githubusercontent.com/github-production-release-asset-2e65be/19708981/e0acf967-faba-4c34-a6ab-f544302e073b?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240412%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240412T023235Z&X-Amz-Expires=300&X-Amz-Signature=913fe54621644d5423990d67bad1b37303bdc302c667f2b9327fa3056c0d3bcf&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=19708981&response-content-disposition=attachment%3B%20filename%3Dgosu-amd64&response-content-type=application%2Foctet-stream
0.729 Resolving objects.githubusercontent.com (objects.githubusercontent.com)... 185.199.109.133, 185.199.111.133, 185.199.110.133, ...
0.765 Connecting to objects.githubusercontent.com (objects.githubusercontent.com)|185.199.109.133|:443... connected.
0.826 Unable to establish SSL connection.
------
failed to solve: process "/bin/sh -c dpkgArch=\"$(dpkg --print-architecture | awk -F- '{ print $NF }')\"  && wget -O /usr/local/bin/gosu \"https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch\"  && chmod +x /usr/local/bin/gosu  && gosu nobody true" did not complete successfully: exit code: 4
```
Reverting to `1.11` solves it. I'm not sure what is the issue here.

Reply:
I ran into a similar issue on Deigo when trying to fetch the bundled SUNDIALS during STEPS installation, so it might have something to do with OIST network.